lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1551,10 +1545,10 @@ AMDMfmaEncodingAttr::getMFMAInstrShapeForOperands(int kWidth, int opIdx) const {
   unsigned nDim = getNDim();
   assert((mDim == nDim) && (mDim == 32 || mDim == 16 || mDim == 4) ||
          (mDim == 64 && nDim == 4) || (mDim == 4 && nDim == 64));
-  constexpr int waveSize = 64; // MFMA is used on wave64 architectures only
+  constexpr int warpSize = 64; // MFMA is used on wave64 architectures only
```

Comment:
Also change the comment

Reply:
This is intentional. `wave64` is a proper noun for the AMDGPU architecture feature, e.g., there's no `warp64`